src/reference_tree.ml

```diff
@@ -20,6 +20,18 @@ type completion_help_type =
     | Script of string [@name "script"]
     [@@deriving yojson]
 
+type doc_hints = {
+        text: string;
+        hint_type: string;
+    } [@@deriving yojson]
+
+type docs = {
+        headline: string;
+        text: string;
+        usageExample: string;
+        hints: doc_hints list;
+    } [@@deriving yojson]
+
```

Comment:
We have not adopted strict formatting yet, but for the moment, I'd suggest consistency in the file: 4 space indent; closing brace aligned with start of definition.